rclcpp/include/rclcpp/parameter.hpp

```diff
@@ -136,6 +138,45 @@ class ParameterVariant
     return value_.bytes_value;
   }
 
+  // The following get_value() variants allow the use of primitive types
+
+  template<typename type>
+  typename std::enable_if<std::is_integral<type>::value && !std::is_same<type, bool>::value,
+  int64_t>::type
```

Comment:
The common rule says: everything with in block (`(`, `{`, `<`) should be indented one level further in order to make it distinguishable.

In this case you can either do:

```
typename std::enable_if<
  std::is_integral<type>::value && !std::is_same<type, bool>::value, int64_t>::type
```

or

```
typename std::enable_if<
  std::is_integral<type>::value && !std::is_same<type, bool>::value,
  int64_t>::type
```